builtin/providers/aws/resource_aws_api_gateway_method.go

```diff
@@ -107,6 +112,7 @@ func resourceAwsApiGatewayMethodCreate(d *schema.ResourceData, meta interface{})
 		RequestModels:     aws.StringMap(models),
 		RequestParameters: aws.BoolMap(parameters),
 		ApiKeyRequired:    aws.Bool(d.Get("api_key_required").(bool)),
+		AuthorizerId:      aws.String(d.Get("authorizer_id").(string)),
```

Comment:
If this is optional, shouldn't we check that it is present before passing it to the API?